cli/cmd/stat.go

```diff
@@ -392,9 +397,12 @@ func printSingleStatTable(stats map[string]*row, resourceType string, w *tabwrit
 				stats[key].tlsPercent * 100,
 			}...)
 
+			if resourceType != k8s.Authority {
```

Comment:
Rather than multiple checks for `k8s.Authority`, what do you think about making it an explicit function...?

```diff
diff --git a/cli/cmd/stat.go b/cli/cmd/stat.go
index 5761b173..1393a213 100644
--- a/cli/cmd/stat.go
+++ b/cli/cmd/stat.go
@@ -320,9 +320,14 @@ func printStatTables(statTables map[string]map[string]*row, w *tabwriter.Writer,
 	}
 }
 
-func showTCPStats(options *statOptions, resourceType string) bool {
+func showTCPConns(resourceType string) bool {
+	return resourceType != k8s.Authority
+}
+
+func showTCPBytes(options *statOptions, resourceType string) bool {
 	return (options.outputFormat == wideOutput || options.outputFormat == jsonOutput) &&
-		resourceType != k8s.Authority
+		showTCPConns(resourceType)
+
 }
 
 func printSingleStatTable(stats map[string]*row, resourceType string, w *tabwriter.Writer, maxNameLength int, maxNamespaceLength int, options *statOptions) {
@@ -343,7 +348,7 @@ func printSingleStatTable(stats map[string]*row, resourceType string, w *tabwrit
 		"TCP CONN",
 	}...)
 
-	if showTCPStats(options, resourceType) {
+	if showTCPBytes(options, resourceType) {
 		headers = append(headers, []string{
 			"READ BYTES/SEC",
 			"WRITE BYTES/SEC",
@@ -361,13 +366,13 @@ func printSingleStatTable(stats map[string]*row, resourceType string, w *tabwrit
 		templateString := "%s\t%s\t%.2f%%\t%.1frps\t%dms\t%dms\t%dms\t%.f%%\t%d\t\n"
 		templateStringEmpty := "%s\t%s\t-\t-\t-\t-\t-\t-\t-\t\n"
 
-		if showTCPStats(options, resourceType) {
+		if showTCPBytes(options, resourceType) {
 			templateString = "%s\t%s\t%.2f%%\t%.1frps\t%dms\t%dms\t%dms\t%.f%%\t%d\t%.1fB/s\t%.1fB/s\t\n"
 			templateStringEmpty = "%s\t%s\t-\t-\t-\t-\t-\t-\t-\t-\t-\t\n"
 		}
 
 		// always show TCP Connections as - for Authorities
-		if resourceType == k8s.Authority {
+		if !showTCPConns(resourceType) {
 			templateString = "%s\t%s\t%.2f%%\t%.1frps\t%dms\t%dms\t%dms\t%.f%%\t-\t\n"
 		}
 
@@ -397,11 +402,11 @@ func printSingleStatTable(stats map[string]*row, resourceType string, w *tabwrit
 				stats[key].tlsPercent * 100,
 			}...)
 
-			if resourceType != k8s.Authority {
+			if showTCPConns(resourceType) {
 				values = append(values, stats[key].tcpOpenConnections)
 			}
 
-			if showTCPStats(options, resourceType) {
+			if showTCPBytes(options, resourceType) {
 				values = append(values, []interface{}{
 					stats[key].tcpReadBytes,
 					stats[key].tcpWriteBytes,
```

Reply:
ah! I like it, it's clearer!